src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,61 @@
+immutable MaxwellBoltzmann <: ContinuousUnivariateDistribution
+    a::Float64 # Scale factor
+    chisqd::Chisq
+
+    function MaxwellBoltzmann(a::Real)
+        a > zero(a) || error("MaxwellBoltzmann: a must be positive")
+        @compat new(Float64(a), Chisq(3.0))
+    end
+end
+
+@distr_support MaxwellBoltzmann 0.0 Inf
+
+#### Parameters
+
+scale(d::MaxwellBoltzmann) = d.a
+params(d::MaxwellBoltzmann) = (d.a,)
+
+#### Statistics
+
+mean(d::MaxwellBoltzmann) = 1.5957691216057308 * d.a # 2*sqrt(2/π)
+median(d::MaxwellBoltzmann) = 1.5381722544550522 * d.a # quantile(d, 0.5)
```

Comment:
`quantile` is probably too hard for the compiler to handle so here the value is required if speed is a high priority for the `median` function.